Added section about contributing. (#432)
observatorium/observatorium: docs/community/README.md

# Community

> Hello and welcome to Observatorium community. Everyone contribution is very valuable.

Reviewer: ```suggestion
Hello and welcome to the Observatorium community. Everyone's contribution is very valuable to us!
```

To interact with the Observatorium community use any of the following channels:

## Issues, Bugs, Ideas, Questions

> For any asynchronous contributions, please fill [github.com/observatorium/observatorium](https://github.com/observatorium/observatorium) GitHub Issue or start Discussion thread.

Reviewer: ```suggestion
For any asynchronous communication, please fill [github.com/observatorium/observatorium](https://github.com/observatorium/observatorium) GitHub Issue or start Discussion thread.
```

Before doing that, make sure to search for any previous relevant conversations.

## Slack

Join our [CNCF `#observatorium` slack](https://cloud-native.slack.com) to meet with community for some synchronous communication.

## Public Community Meetings

> If you want to learn on what is happening with the community efforts or you want to discuss anything relevant, meet us online! We are holding a meeting every second Wednesday 15:00 CET on Google Meet.

Reviewer: ```suggestion
If you want to learn about what is happening with the community efforts or you want to discuss anything relevant, meet us online! We are holding a meeting every second Wednesday 15:00 CET on Google Meet.
```
